### PR TITLE
feat: add test hint for grant moderator availability

### PR DESCRIPTION
**Pull request by gimre**

### CODE REVIEW

File: react/features/base/testing/components/TestConnectionInfo.js [hunk 1]
```diff
@@ -204,6 +215,7 @@ class TestConnectionInfo extends Component<Props, State> {
  * @returns {{
  *     _conferenceConnectionState: string,
  *     _conferenceJoinedState: string,
+ *     _grantModeratorEnabled: string,
```
Comment (saghul) [inline react/features/base/testing/components/TestConnectionInfo.js:218]: Remove all of there and say {Props} please.

File: react/features/base/testing/components/TestConnectionInfo.js [hunk 2]
```diff
@@ -184,6 +192,9 @@ class TestConnectionInfo extends Component<Props, State> {
                 <TestHint
                     id = 'org.jitsi.meet.conference.joinedState'
                     value = { this.props._conferenceJoinedState } />
+                <TestHint
+                    id = 'org.jitsi.meet.conference.grantModeratorEnabled'
+                    value = { this.props._grantModeratorEnabled } />
```
Comment (saghul) [inline react/features/base/testing/components/TestConnectionInfo.js:197]: Just say true here. Any version of this frontend will run with an up to date enough lib-jitsi-meet, so you can just say true here.

File: react/features/base/testing/components/TestConnectionInfo.js [hunk 3]
```diff
@@ -184,6 +192,9 @@ class TestConnectionInfo extends Component<Props, State> {
                 <TestHint
                     id = 'org.jitsi.meet.conference.joinedState'
                     value = { this.props._conferenceJoinedState } />
+                <TestHint
+                    id = 'org.jitsi.meet.conference.grantModeratorEnabled'
```
Comment (saghul) [inline react/features/base/testing/components/TestConnectionInfo.js:196]: Can you please call this `org.jitsi.meet.conference.grantModeratorAvailable` ? There is no way to disable it so I think it makes a bit more semantic sense.

File: react/features/base/testing/components/TestConnectionInfo.js (hunk 2 above)
Comment (gimre) [reply react/features/base/testing/components/TestConnectionInfo.js]: Okai

File: react/features/base/testing/components/TestConnectionInfo.js (hunk 3 above)
Comment (gimre) [reply react/features/base/testing/components/TestConnectionInfo.js]: Updated! :D

File: react/features/base/testing/components/TestConnectionInfo.js (hunk 1 above)
Comment (gimre) [reply react/features/base/testing/components/TestConnectionInfo.js]: done!

File: react/features/base/testing/components/TestConnectionInfo.js (hunk 2 above)
Comment (gimre) [reply react/features/base/testing/components/TestConnectionInfo.js]: done!

File: react/features/base/testing/components/TestConnectionInfo.js [hunk 4]
```diff
@@ -216,6 +228,7 @@ function _mapStateToProps(state) {
     return {
         _conferenceConnectionState: state['features/testing'].connectionState,
         _conferenceJoinedState: conferenceJoined.toString(),
+        _grantModeratorAvailable: typeof APP.conference._room.grantOwner === 'function',
```
Comment (saghul) [inline react/features/base/testing/components/TestConnectionInfo.js:231]: i still see this logic here, forgot to push?

Comment (gimre) [reply react/features/base/testing/components/TestConnectionInfo.js:231]: github is being weird, lemme close and re-open :s

**Pull request closed without merging**